Error while checking build/O2/fullCI for a664427ef99370a88ef704c87d544d2e1e8b7a33 at 2025-03-13 14:55:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'


## sw/BUILD/O2Physics-latest/log
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1574:82: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1574:82: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1574:82: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1574:82: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1574:82: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
/sw/slc8_x86-64/O2/14057-slc8_x86-64-local1/include/Framework/ASoA.h:1508:96: error: 'const struct o2::framework::Entry' has no member named 'second'
[0 more errors; see full log]
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/14057/a664427ef99370a88ef704c87d544d2e1e8b7a33/build_O2_fullCI/pretty.html).

--- a/Framework/Core/include/Framework/ASoA.h
+++ b/Framework/Core/include/Framework/ASoA.h
@@ -1505,7 +1505,7 @@ auto doSliceBy(T const* table, o2::framework::PresliceBase<C, Policy, OPT> const
 {
   if constexpr (OPT) {
     if (container.isMissing()) {
-      missingOptionalPreslice(getLabelFromType<std::decay_t<T>>().data(), container.bindingKey.second.c_str());
+      missingOptionalPreslice(getLabelFromType<std::decay_t<T>>().data(), container.bindingKey.key.c_str());
     }
   }
   uint64_t offset = 0;
@@ -1542,7 +1542,7 @@ auto doSliceBy(T const* table, o2::framework::PresliceBase<C, Policy, OPT> const
 {
   if constexpr (OPT) {
     if (container.isMissing()) {
-      missingOptionalPreslice(getLabelFromType<std::decay_t<T>>().data(), container.bindingKey.second.c_str());
+      missingOptionalPreslice(getLabelFromType<std::decay_t<T>>().data(), container.bindingKey.key.c_str());
     }
   }
   auto selection = container.getSliceFor(value);
@@ -1571,7 +1571,7 @@ auto doFilteredSliceBy(T const* table, o2::framework::PresliceBase<C, framework:
 {
   if constexpr (OPT) {
     if (container.isMissing()) {
-      missingOptionalPreslice(getLabelFromType<T>().data(), container.bindingKey.second.c_str());
+      missingOptionalPreslice(getLabelFromType<T>().data(), container.bindingKey.key.c_str());
     }
   }
   uint64_t offset = 0;

